src/legacy/ui/public/new_platform/new_platform.ts

```diff
@@ -187,6 +187,18 @@ export const legacyAppRegister = (app: App<any>) => {
         ),
         onAppLeave: () => undefined,
       };
+
+      if (app.updater$) {
```

Comment:
I noticed that nothing was being done with the app.updater$ here in the `legacyAppRegister` function so I added this block. It's the same thing that's happening here: https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/public/local_application_service/local_application_service.ts/#L89